app/scripts/modules/core/src/deploymentStrategy/strategies/redblack/redblack.strategy.ts

```diff
@@ -28,7 +28,7 @@ DeploymentStrategyRegistry.registerStrategy({
       rollback: {
         onFailure: false,
       },
-      maxRemainingAsgs: !command.strategy ? 2 : command.maxRemainingAsgs,
```

Comment:
Can you verify that this Boolean logic is correct?

Reply:
The old logic? I imagine it was there to preserve the value if someone were to switch strategies then switch back

Reply:
Oh but I suppose `!command.strategy` was probably never very correct to start. `command.maxRenainingAsgs === undefined` is probably more appropriate